cockatrice/src/interface/widgets/tabs/api/archidekt/api_response/deck/archidekt_api_response_deck.h

```diff
@@ -12,6 +12,228 @@
 #include <QString>
 #include <QVector>
 
+namespace ArchidektFormats
+{
+enum class DeckFormat
+{
+    // The order is important since archidekt returns an integer
+    Standard,
```

Comment:
If the exact integer of the enum is important, then it might be better to make it explicit
```suggestion
    Standard = 1,
```